### ISSUE TITLE
Release 1.2.0 doesn't ship with styles folder

### ISSUE THREAD

**Issue by Jackymancs4** [da2x/instapaper-webextension#3]
Hello @da2x 
The current release package does not contain the `styles` folder and the addon is not styled at all .
I've noticed it a while ago but never found time to investigate.

**Comment by da2x** [comment]
How the heck can I possibly have missed that!? 🤯I use this several times a day.

Thanks for the report. I’ll push out a new version right away.

**Comment by Jackymancs4** [comment]
Hey, thanks for the quick new release.
Unluckily, release 1.2.1 also lack of the `styles` folder, making this still an issue. You can quickly see that [here](https://robwu.nl/crxviewer/?crx=https%3A%2F%2Faddons.mozilla.org%2Fit%2Ffirefox%2Faddon%2Finstapaper-tools%2F).

If you have always used the newly added makefile, the problem might be
https://github.com/da2x/instapaper-webextension/blob/d4975ddeb2089bff7502e945c5977a4d85a34504/Makefile#L6

being 

```
SOURCES += $(shell find styles  -name '*.cs')
```

instead of

```
SOURCES += $(shell find styles  -name '*.css')
```

Not sure though, as I haven't tested it.

**Comment by da2x** [comment]
Fixed in commit 9326892c2a3207fc80ee65bb8907d2e73cc15cfc. The build script should prevent this from happening again. Thanks again for checking my work!
